### ISSUE TITLE
[jcm] Font lisibility

### ISSUE THREAD

**Issue by Sandra97** [joomla/joomla-websites#849]
I don't know if it's only me because I'm getting old, but the text font is really hard to read as it looks bold (and result, the `<strong>` elements are highlighted anymore). 
Lisibility is better at 13px, and strong elements reappear. Or you can just use the template css without the override, it solves this issue too.

Thank you!

**Comment by JacquesR** [comment]
I don't observe the same issue, but that's most likely due to the old font looking different on different devices and/or OS.

![font-issue-old](https://cloud.githubusercontent.com/assets/3589025/22929609/48d834d2-f2c5-11e6-89de-c1fe1e98b900.png)

I have now updated the CSS to use the default template font, so it should now look more similar on all devices.

![font-issue-new](https://cloud.githubusercontent.com/assets/3589025/22929636/5e10f7f8-f2c5-11e6-8590-2cfa084c1847.png)

I did opt to keep the font size at 14px and line-height at 1.5 for easier reading.
The old custom CSS will get more cleaning up at a later stage.
